CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
+### Added
```

Comment:
I just remembered, in the Changelog specs (https://keepachangelog.com/en/1.0.0/) it does say that there is a heading for "Security". That would probably be better to use here instead of putting it under "Added".

Reply:
Done! :)